src/components/TextParam.tsx

```diff
@@ -94,6 +94,7 @@ const ParamsBox: FunctionComponent = () => {
 				<button onClick={onAddLanguage}>Add language</button>
 			</div>
 		</Fragment>
+		<label>static: <input type="checkbox" checked={selectedEntity.params.isStatic} onChange={onToggleStatic}/></label>
```

Comment:
No need for this, `<StaticParam/>` already does it.

Reply:
Alright, I wasn’t sure if it would be needed. Does this just add an extra static checkbox?

Reply:
Yep. And it would break when you click it because `onToggleStatic` doesn't exist.

Reply:
oh yeah since it’s in another file.